src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7412,6 +7412,7 @@ public partial class FileStream : System.IO.Stream
     public static partial class Directory
     {
         public static System.IO.DirectoryInfo CreateDirectory(string path) { throw null; }
+        public static System.IO.FileSystemInfo CreateSymbolicLink(string path, string pathToTarget) { throw null; }
```

Comment:
@adamsitnik as mentioned in https://github.com/dotnet/runtime/issues/24271#issuecomment-850164852, in the common case, the user won't care about the instance that gets returned here and this method can/should return `void` instead.

Reply:
@tmds  Most probably you are right, but in such case users can just ignore the result returned from the method (it's not disposable).

```cs
CreateSymbolicLink("a", "b");
```